ChatQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -344,6 +325,53 @@ docker compose -f compose_vllm.yaml up -d
 
 ### Validate Microservices
 
+Note, before verify the microservices by curl or API, please make sure the ports of the microservices are opened in the firewall of the cloud node.
+
+> See one example below. Please open up these ports in the EC2 instance based on the IP addresses you want to allow:
+
+```
+redis-vector-db
```

Comment:
remove the long list,  just  refer to ## Network Port & Security

Reply:
## Network Port & Security refer to this chapter too.
Here is the port list for microservices.
It's very useful to user when open ports.